plugin/lion.vim

```diff
@@ -2,6 +2,8 @@
 " Maintainer:   Tom McDonald <http://github.com/tommcdo>
 " Version:      1.0
 
+let b:lion_squeeze_spaces = get(b:, 'lion_squeeze_spaces', 0)
```

Comment:
We can't rely on one-time initialization for a buffer-local variable, as a new instance of it is created for each buffer (which could happen long after Vim initialization). Instead, we should use this `get()` call in the `if`-statement.

I had an idea to check the buffer variable first, then fall back to the global variable, then to a default value. What do you think?